cmd/root.go

```diff
@@ -163,6 +165,107 @@ func parseToolsFile(ctx context.Context, raw []byte) (ToolsFile, error) {
 	return toolsFile, nil
 }
 
+func parse(ctx context.Context, buf []byte, logger log.Logger) {
+	logger.DebugContext(ctx, "Attempting to parse updated tools file.")
+	// time.Sleep(5 * time.Second)
+	// logger.DebugContext(ctx, "Parse finished.")
+	// TODO: add logic for parsing
+}
+
+// watchFile checks for changes in the provided yaml tools file.
+func watchFile(toolsFileName string, ctx context.Context, logger log.Logger) {
```

Comment:
nit: ctx should be first argument.

Can we extract logger from context? I think we might be doing this in other places.

Reply:
I don't think it should happen (if the logger is already created it should be in the context) but if it does we probably need to panic